provision/acc_provision/acc_provision.py

```diff
@@ -151,6 +181,8 @@ def deep_merge(user, default):
 def config_default():
     # Default values for configuration
     default_config = {
+        "template_type": TEMPLATE_KUBE,
+        "template_generator": generate_kube_yaml,
```

Comment:
Putting a function in here is asking for trouble since the user can set this parameter in their input YAML file, at which point presumably we just blindly call the string as a function.